index.js

```diff
@@ -20,9 +21,9 @@ function inCommand (cmd) {
 }
 
 exports.inPublish = function () {
-  return inCommand(/^pu(b(l(i(sh?)?)?)?)?$/)
+  return inCommand(/^(?:pre)?pu(b(l(i(sh?)?)?)?)?$/)
 }
 
 exports.inInstall = function () {
-  return inCommand(/^i(n(s(t(a(ll?)?)?)?)?)?$/)
+  return inCommand(/^(?:pre)?i(n(s(t(a(ll?)?)?)?)?)?$/)
```

Comment:
Wait, you want the bundled `in-install` script to be true if you ran `npm run preinstall`? But... why would you have an `in-install` script for the `preinstall` lifecycle? That only has one way of being run, so it would always be true…